tests/test_fiod.sh

```diff
@@ -27,7 +27,7 @@ function functional_test_fio {
   wait_for "kubectl -n my-ripsaw wait --for=condition=Initialized -l app=fio-benchmark-$uuid pods --timeout=300s" "300s"
   fio_pod=$(get_pod "app=fiod-client-$uuid" 300)
   wait_for "kubectl wait --for=condition=Initialized pods/$fio_pod -n my-ripsaw --timeout=500s" "500s" $fio_pod
-  wait_for "kubectl wait --for=condition=complete -l app=fiod-client-$uuid jobs -n my-ripsaw --timeout=500s" "500s" $fio_pod
+  wait_for "kubectl wait --for=condition=complete -l app=fiod-client-$uuid jobs -n my-ripsaw --timeout=900s" "900s" $fio_pod
```

Comment:
Any idea why this PR would cause a delay here? The 500s timeout seemed to work prior. My concern going to 900s is that it would be done potentially twice in this test and then repeated up to 3 times. Any way to scale this down some?